lib/preload.js

```diff
@@ -498,6 +507,13 @@ class Preloader extends EventEmitter {
     return `${count} ${thing}${(count !== 1) ? 's' : ''}`
   }
 
+  _deviceTypeArch (deviceType) {
+    return _.find(this.deviceTypes, (dt) => {
+      return (dt.slug === deviceType)
+    })
+    .arch
```

Comment:
This doesn't handle not finding the device type, and the error shown wouldn't be that great (`cannot find arch of undefined`)

Reply:
Good catch, updated.